extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -524,8 +545,8 @@ public static function getHostById( $cityId ) {
 		$hostName = \WikiFactory::getVarValueByName( 'wgServer', $cityId );
 
 		if ( !empty( $wgDevelEnvironment ) ) {
-			if ( strpos( $hostName, "wikia.com" ) ) {
-				$hostName = str_replace( "wikia.com", "{$wgDevelEnvironmentName}.wikia-dev.com", $hostName );
+			if ( strpos( $hostName, self::WIKIA_COM ) ) {
+				$hostName = str_replace( self::WIKIA_COM, "{$wgDevelEnvironmentName}.wikia-dev.com", $hostName );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)